openreview/conference/templates/programchairWebfield.js

```diff
@@ -1676,6 +1687,154 @@ var displayPaperStatusTable = function() {
 
 };
 
+var displayRejectedWithdrawnPaperStatusTable = function () {
```

Comment:
Withdrawn and desk rejected submission may have reviews, reviewers or AC assigned? can we just use the same table "Paper Status"?